app-emulation/qemu/qemu-2.11.0.recipe

```diff
@@ -0,0 +1,179 @@
+	cmd:qemu_system_mipsel$commandSuffix = $portVersion
+	cmd:qemu_system_moxie$commandSuffix = $portVersion
+	cmd:qemu_system_or32$commandSuffix = $portVersion
+	cmd:qemu_system_ppc$commandSuffix = $portVersion
+	cmd:qemu_system_ppc64$commandSuffix = $portVersion
+	cmd:qemu_system_ppcemb$commandSuffix = $portVersion
+	cmd:qemu_system_s390x$commandSuffix = $portVersion
+	cmd:qemu_system_sh4$commandSuffix = $portVersion
+	cmd:qemu_system_sh4eb$commandSuffix = $portVersion
+	cmd:qemu_system_sparc$commandSuffix = $portVersion
+	cmd:qemu_system_sparc64$commandSuffix = $portVersion
+	cmd:qemu_system_unicore32$commandSuffix = $portVersion
+	cmd:qemu_system_x86_64$commandSuffix = $portVersion
+	cmd:qemu_system_xtensa$commandSuffix = $portVersion
+	cmd:qemu_system_xtensaeb$commandSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	cmd:iasl$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libcapstone$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libfdt$secondaryArchSuffix
+	lib:libgcrypt$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgnutls$secondaryArchSuffix
+	lib:libgpg_error$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:liblzo2$secondaryArchSuffix
+	lib:libncursesw$secondaryArchSuffix
+	lib:libnettle$secondaryArchSuffix
+	lib:libpixman_1$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libsdl2_2.0$secondaryArchSuffix
+	lib:libsnappy$secondaryArchSuffix
+	lib:libssh2$secondaryArchSuffix
+	lib:libssp$secondaryArchSuffix
+	lib:libtasn1$secondaryArchSuffix
+	lib:libusb_1.0$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	devel:libbz2$secondaryArchSuffix
+	devel:libcapstone$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libfdt$secondaryArchSuffix
+	devel:libgcrypt$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgnutls$secondaryArchSuffix
+	devel:libgpg_error$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:liblzo2$secondaryArchSuffix
+	devel:libncursesw$secondaryArchSuffix
+	devel:libnettle$secondaryArchSuffix
+	devel:libpixman_1$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libsdl2_2.0$secondaryArchSuffix
+	devel:libsnappy$secondaryArchSuffix
+	devel:libssh2$secondaryArchSuffix
+	devel:libtasn1$secondaryArchSuffix
+	devel:libusb_1.0$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:awk
+	cmd:cmp
+	cmd:gcc$secondaryArchSuffix
+	cmd:gcov$secondaryArchSuffix
+	cmd:gprof$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:find
+	cmd:iasl$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:pod2man
+	cmd:python
+	"
+
+defineDebugInfoPackage qemu$secondaryArchSuffix \
+	"$commandBinDir"/qemu-system-x86_64 \
```

Comment:
```diff
 defineDebugInfoPackage qemu$secondaryArchSuffix \
-	"$commandBinDir"/qemu-system-x86_64 \
-	"$commandBinDir"/qemu-system-i386
+	"`echo "$PROVIDES" | sed -n \
+		-e "s|cmd:\(.*\)$commandSuffix = .*|$commandBinDir/\1|p" \
+	`"
```

Reply:
No, provides have more cmd Than in the package.

Reply:
The above comment assumes `PROVIDES` is defined with [this suggestion](#discussion_r167442676).
(But I need to fix the sed command above.)

Reply:
> No, provides have more cmd Than in the package.

I have built qemu on x86_64, and, not counting the qemu symlink, the only runtimes I get in `/system/bin/` are:
```
qemu-img
qemu-io
qemu-system-i386
qemu-system-x86_64
```

I guess the other `qemu-system-*` need some extra dependencies to be built.
Do you obtain other runtimes (which I did not get)?
(If not, we should either drop the extra `cmd:` lines or comment them out.)

Reply:
No, but korli and mmlr worked on this recipe, they defined it so, and i will do everything, but i won't do that.
The recipe is still disabled on every arch and it have almost no practical use, so i don't want to change the provides sections.

Reply:
OK, no problem.
(I did not know that the `cmd:` lines were from korli and mmlr.)
So, of course, that's OK to keep them.